File: charcorba-c++/Client/src/pixel.cpp
#include <pixel.h>
#include <QPoint>
#include <QColor>

Pixel::Pixel()
{
	
}

Pixel::~Pixel()
{
	
}

Pixel::Pixel(QPoint qpoint, 
	         QColor qcolor,
	         bool est_continu)
{
	m_qpoint = qpoint;
	m_qcolor = qcolor;
	m_est_continu = est_continu ;
}

File: trunk/charcorba-java/tableau_blanc/Pixel.java
package tableau_blanc;

import org.eclipse.swt.graphics.Color;
import org.eclipse.swt.graphics.Point;

public class Pixel {
		
		public Point m_point;
		public Color m_color;
		public  boolean m_est_continu;
		
		public Pixel(Point point, Color color, boolean est_continu)
		{
			m_point = point;
			m_color = color;
			m_est_continu = est_continu;
		}
}


